### PR TITLE
[fix](move-memtable) change brpc connection type to single

### PR DESCRIPTION
**Pull request by kaijchen**
## Proposed changes

Previously, we use "pooled" connection to distinguish streaming connections
with non-streaming connections.

According to brpc docs [connection-type][1]:

> Max number of pooled connections from one client to one server is limited
> by -max_connection_pool_size.
>
> Note the number is not same as “max number of connections”.
> New connections are always created when there’s no idle ones in the pool;
> the returned connections are closed immediately when the pool already
> has max_connection_pool_size connections.

If there are too many connections, the connection will be closed immediately
once the initial `open_load_stream` rpc has finished. Causing the stream
to be closed in the meantime. Subsequent writes to the stream will encounter
"StreamWrite failed, err=22".

This PR changes brpc connection type to `"single"` for streaming connections.
And set brpc connection group to `"streaming"` to distinguish connections.

[1]: https://brpc.apache.org/docs/client/basics/#connection-type

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

File: be/src/vec/sink/load_stream_stub.cpp [hunk 1]
```diff
@@ -175,9 +175,9 @@ Status LoadStreamStub::open(BrpcClientCache<PBackendService_Stub>* client_cache,
         *request.add_tablets() = tablet;
     }
     POpenLoadStreamResponse response;
-    // use "pooled" connection to avoid conflicts between streaming rpc and regular rpc,
-    // see: https://github.com/apache/brpc/issues/392
-    const auto& stub = client_cache->get_new_client_no_cache(host_port, "baidu_std", "pooled");
+    // set load_id as connection_group to distinguish different streaming connections
+    const auto& stub = client_cache->get_new_client_no_cache(host_port, "baidu_std", "single",
+                                                             print_id(_load_id));
```
Comment (kaijchen) [inline be/src/vec/sink/load_stream_stub.cpp:180]:
```suggestion
                                                             "streaming");
```

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Pull request merged**